### PR TITLE
Portfolio: setup and mobile first

### PR DESCRIPTION
**Pull request by jacobrees**
This pull request is for the implementation of a navbar and headline for this portfolio website.

The main features include:

- A fully responsive navbar
- A fully responsive headline section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Microverse Portfolio</title>
+  </head>
+  <body>
+    <nav class="navbar">
+        <div class="navbar-wrapper">
+            <h2 class="navbar-logo">My Logo</h2>
```
Comment (brainconnect93) [inline index.html:18]: 1. I would kindly suggest you make the logo be wrap inside of an anchor tag `<a>`. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+              <a class="nav-link" href="#">Portfolio</a>
+              <a class="nav-link" href="#">About</a>
+              <a class="nav-link" href="#">Contact</a>
+              <a class="nav-link"href="#"><img src="imgs/mail.svg" alt="Mail Link"></a>
+              <img
+                class="hamburger-menu"
+                src="imgs/hamburger.svg"
+                alt="Hamburger Menu"
+              />
+            </div>
```
Comment (brainconnect93) [inline index.html:29]: - [ ] Kindly make this part of the `header` be wrap inside of a `nav` tag. The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Microverse Portfolio</title>
+  </head>
+  <body>
+    <nav class="navbar">
```
Comment (brainconnect93) [inline index.html:16]: - [ ] I would kindly suggest you make the container of this part to be a `header` tag that will be accommodating both the `a` and the `nav` tag

E.G:
```HTML
<header>
   <a href="#">My Logo</a>
   <nav>
      <img class="hamburger-menu" src="imgs/hamburger.svg" alt="Hamburger Menu" />
   </nav>
</header>
```

The container of the section is a `header` tag.

* Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hesitate to contact me.
+        </p>
+        <div class="headline-links">
+            <a class="headline-link" href="#"><img class="headline-link-img" src="imgs/github.svg" alt="GitHub Link"></a>
+            <a class="headline-link" href="#"><img class="headline-link-img" src="imgs/linkedin.svg" alt="Linkedin Link"></a>
+            <a class="headline-link" href="#"><img class="headline-link-img" src="imgs/angelist.svg" alt="Angelist Link"></a>
+            <a class="headline-link" href="#"><img class="headline-link-img" src="imgs/twitter.svg" alt="Twitter Link"></a>
+            <a class="headline-link" href="#"><img class="headline-link-img" src="imgs/medium.svg" alt="Medium Link"></a>
+        </div>
```
Comment (brainconnect93) [inline index.html:48]: - [ ] I would kindly suggest you make the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).
   - Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

E.G:
```HTML
<ul>
  <li><a class="headline-link" href="#"><img class="headline-link-img" src="imgs/github.svg" alt="GitHub Link"></a></li>
  <li><a class="headline-link" href="#"><img class="headline-link-img" src="imgs/linkedin.svg" alt="Linkedin Link"></a></li>
  <li><a class="headline-link" href="#"><img class="headline-link-img" src="imgs/angelist.svg" alt="Angelist Link"></a></li>
  <li><a class="headline-link" href="#"><img class="headline-link-img" src="imgs/twitter.svg" alt="Twitter Link"></a></li>
  <li><a class="headline-link" href="#"><img class="headline-link-img" src="imgs/medium.svg" alt="Medium Link"></a></li>
</ul>
```

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @jacobrees, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 

- Descriptive Pull Request title and summary. ✔️  
- Portfolio: setup and mobile first ✔️  
- Follow the Figma Design Template ✔️  
- Touchable mobile screen size of `375px` ✔️  
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ABDULALI3468 — APPROVED** [review]
Hi @jacobrees , 👍 

# APPROVED 💯 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 
☑️Implemented instructions required by the previous reviewer.
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**